glancesync/glancesync_serversfacade.py

```diff
@@ -118,7 +183,23 @@ def delete_image(self, regionobj, id, confirm=True):
         :param confirm: ask for confirmation
         :return: true if image was deleted, false if it was canceled by user
         """
-        return _delete_image(regionobj, id, confirm)
+        client = self._get_glanceclient(regionobj.region)
+        if confirm:
+            confirm = raw_input('Delete image {0}? [y/N]'.format(id))
```

Comment:
Wait, if we are moving to a server solution in which we execute remotely this glancesync, can I assume that the confirm option should be False?

If it is true we have to write the corresponding comment here to say it.